warehouse/query-core/src/main/java/datawave/query/config/ShardQueryConfiguration.java

```diff
@@ -515,6 +515,18 @@ public class ShardQueryConfiguration extends GenericQueryConfiguration implement
      */
     private double fieldIndexHoleMinThreshold = 1.0d;
 
+    /**
+     * The set of date types that, if the query's end date is the current date, will NOT result in any date range adjustments or the addition of a
+     * SHARDS_AND_DAYS hint.
+     */
+    private Set<String> noExpansionIfCurrent = Collections.emptySet();
```

Comment:
Can this be renamed to denote it is a list of date types somehow?  Perhaps noExpansionIfCurrentDateTypes

Reply:
Done.